.github/workflows/review-scalingo.yml

```
@@ -27,3 +27,15 @@ jobs:
         run: |
           PATH=$HOME/bin:$PATH
           ./scripts/link-scalingo-apps.sh
+  add-pull-request-comment:
+    name: Add review app URL to Pull Request
+    runs-on: ubuntu-latest
+    needs: [scalingo-link]
+    steps:
+      - name: Add review app URL
+        uses: blablacar/action-sticky-description@master
```

Important : utiliser un SHA1 de commit pour référencer une action tierce partie afin de ne pas s'exposer à une _supply chain attack_.
```suggestion
        uses: blablacar/action-sticky-description@df5650c838f2bbc860ab8edd059b6e255fd7bb82
```